app/controllers/places_controller.rb

```diff
@@ -10,61 +9,46 @@ def new
     @place = Place.new
   end
 
-def create
-  @place = current_user.places.create(place_params)
-  if @place.valid?
-    redirect_to root_path
-  else
-    return render :new, status: :unprocessable_entity
-  end
-
-  def place_params
```

Comment:
`place_params` is defined at the bottom of the file.  This is a second declaration and not needed.